tests/test_flow/test_protocols.py

```diff
@@ -196,6 +199,80 @@ def df():
     pdt.assert_frame_equal(builder.build().get('df'), df_value)
 
 
+def test_simple_dask_dataframe(builder):
+    df_value = df_from_csv_str('''
+    color,number
+    red,1
+    blue,2
+    green,3
+    ''')
+    dask_df = dd.from_pandas(df_value, npartitions=1)
```

Comment:
Should we have a test case with multiple partitions?

Reply:
Definitely, adding!